<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dakar 2015</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>391</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>411</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConvert">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionFile"/>
     <addaction name="actionFolder"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuParametres">
    <property name="title">
     <string>Parametres</string>
    </property>
    <addaction name="actionDirectoryPath"/>
    <addaction name="actionCategory"/>
    <addaction name="actionEditZone"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuParametres"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionFolder">
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
  <action name="actionDirectoryPath">
   <property name="text">
    <string>Directory Path</string>
   </property>
  </action>
  <action name="actionCategory">
   <property name="text">
    <string>Category</string>
   </property>
  </action>
  <action name="actionEditZone">
   <property name="text">
    <string>Edit Zone</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
